data_steward/cdr_cleaner/cleaning_rules/update_invalid_zip_codes.py

```diff
@@ -12,6 +10,7 @@
         value_as_string and value_source_value = 'Response removed due to invalid value'
         value_as_number = 0
         value_source_concept_id = 2000000010
+zip3_lookup is a view that uses "Census Bureau US Boundaries" public dataset available in BigQuery.
```

Comment:
I found this Lauren's comment in `DC-2739`(the investigation ticket). Should we update the logic as suggested? Let me know if I am missing any discussions in the background.

```
- #####
- #########
- #####-####

~ the work for DC-2727 be changed to ignore that zip code table entirely and focus on meeting a regular expression criteria.
```

Reply:
I left a comment in DC-2739 on this subject. I'll start work on this again when I have a reply. I believe the subject is being discussed.

Reply:
The question of whether to use a zip code lookup table or otherwise will be discussed in DC-2802. For this CR we will continue using zip3_lookup.